docs/vector.rst

```diff
@@ -101,6 +101,51 @@ As a temporary local variable
 
     TODO: add descriptions about ``a(i, j)``
 
+
+XYZW vector component accessors
++++++++++++++++++++++++++++++++
+
+We also provide four handy accessors for the first four vector components:
+
+.. attribute:: a.x
+
+   Same as ``a[0]``.
+
+.. attribute:: a.y
+
+   Same as ``a[1]``.
+
+.. attribute:: a.z
+
+   Same as ``a[2]``.
+
+.. attribute:: a.w
+
+   Same as ``a[3]``.
+
+.. note::
+
+   XYZW accessors can be used for both read and write::
+
+      v = ti.Vector([2, 3, 4])
+      print(v.x)  # 2
+      print(v.y)  # 3
+      print(v.z)  # 4
+      v.y = 8
+      print(v.y)  # 8
+
+   XYZW accessors can be used in both Taichi-scope and Python-scope.
+
+   XYZW accessors don't work for ``ti.Matrix``.
```

Comment:
Can this XYZW accessors be used in both `ti.Vector` and `ti.field`, except `ti.Matrix`? ;)

Reply:
Yes XYZW accessors can be used in both `ti.Vector` and `ti.Vector.field`.